payment/order.go

```diff
@@ -18,24 +19,27 @@ type Order struct {
 	UpdatedAt  time.Time       `json:"updatedAt" db:"updated_at"`
 	TotalPrice decimal.Decimal `json:"totalPrice" db:"total_price"`
 	MerchantID string          `json:"merchantId" db:"merchant_id"`
+	Location   sql.NullString  `json:"location" db:"location"`
```

Comment:
I am concerned that this will not serialize properly to the string json type.  I think it might serialize this way:
```json
location: { "String": "the location", "Valid": true}
```

Reply:
https://github.com/brave-intl/bat-go/pull/308/commits/0e83e99201566e18958932c8439cd7224ec3bc54